src/content/docs/concepts/programs.mdx

```diff
@@ -41,3 +41,17 @@ A Membrane program contains at least three files:
 3. **`memconfig.lock`:** tracks versions of nodes in your graph. It's also automatically generated, and you should never have to edit it manually
 
 Your program's directory may also include a `README.md`, `package.json` with npm dependencies, `.git`, and whatever other helpers and structure you'd like.
+
+## Stopping and deleting programs
+
+To stop a running program, right click the program in the Membrane Explorer and select `Kill` from the dropdown menu. A program must be stopped in order to delete it.
```

Comment:
Maybe we should go ahead and update gaze to say `Stop` instead of `Kill`?

Reply:
Yeah, I was thinking the same. Created a follow-up ticket for myself:
[T-827](https://membrane.height.app/T-827) Update gaze dropdown to Stop instead of Kill